src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar calendar--start-day--sun calendar--month-length--30">
```

Comment:
Your project should work correctly (have 31 days and start on Monday) with invalid modifier values, like `start-day--ghy` or `days--27`. [Checklist p.10](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md)
<img width="1362" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/28480057/58a521d0-416d-4104-8ff2-ffac59b624a4">